tests/cluster/tests/21-many-slot-migration.tcl

```diff
@@ -3,8 +3,11 @@
 source "../tests/includes/init-tests.tcl"
 source "../tests/includes/utils.tcl"
 
+# TODO: This test currently runs without replicas, as failovers (which may
+# happen on lower-end CI platforms) are still not handled properly by the
+# cluster during slot migration.
 test "Create a 10 nodes cluster" {
-    create_cluster 10 10
+    create_cluster 10 0
```

Comment:
so this test didn't at all intend to test replicas and failovers, right?
i.e. normally they don't even happen in this test (happen only on MacOS / FreeBSD CI)?

Reply:
Correct

Reply:
But replicas could introduce some instability that should be handled as well. Couldn't they?

Cluster should be tested in production mode. No one runs production cluster without replicas.